src/form-data-wrapper.js

```diff
@@ -98,10 +98,14 @@ export class FormDataWrapper {
     }
 
     const fieldEntries = [];
-    Object.keys(this.fieldValues_).forEach(name => {
-      const values = this.fieldValues_[name];
-      values.forEach(value => fieldEntries.push([name, value]));
-    });
+    // The following condition should never be false if there's no native
+    // entries(), but the type checker can't infer that.
+    if (this.fieldValues_) {
```

Comment:
We typically do `dev().assert(valueThatShouldExist)` for this.

Reply:
Uh okay. For some reason, the type checker still doesn't recognize that the value is non-null after `assert()`. It throws the same error whether I have 
```
Object.keys(dev().assert(this.fieldValues_))...
````
or
```
dev().assert(this.fieldValues_);
Object.keys(this.fieldValues_)...
```

I have to do a type cast like this for it to work... Any idea?

Reply:
Ah I forgot that only our other `assert` methods return non-null. Either way works.